xla/xla.proto

```diff
@@ -1030,6 +1030,13 @@ message DebugOptions {
   }
   PGLEStrictnessLevel xla_gpu_pgle_accuracy_checker = 341;
 
+  // Enable the scatter determinism expander, an optimized pass that
+  // rewrites scatter operations to ensure deterministic behavior with high
+  // performance.
+  // Note that even when this flag is disabled, scatter operations may still
+  // be deterministic, although with additional overhead.
+  bool xla_gpu_enable_scatter_determinism_expander = 342;
```

Comment:
Actually, it seems this is already used by xla_gpu_experimental_stream_annotation
Please use 343 and update Next id: 344

Reply:
Changed